app.js

```diff
@@ -94,37 +105,18 @@ form.addEventListener('submit', (e) => {
 
     displayThreeProducts();
 
-});
-
-function reset() {
-    initializeState();
-}
-
-function getRandomProduct(someProducts) {
-    const randomIndex = Math.floor(Math.random() * someProducts.length);
-    cont randomProduct = productsData[randomIndex];
-
-    return randomProduct;
-
-}
-
-
-
-
-
-
-
 
+    function reset() {
+        initializeState();
+    }
 
+    function getRandomProduct(someProducts) {
```

Comment:
When I went through your code it seems like this function isn't running, so I would try and troubleshoot why through using console.logs to see where the problem is.